imports/ui/unit/unit.jsx

```diff
@@ -83,12 +143,84 @@ class Unit extends Component {
       })
     }
   }
+  handleReportFilterClicked = (event, index, statusFilterValues) => {
+    this.setState({
+      statusFilterValues: statusFilterValues
+    })
+  }
+
+  handleReportSortClicked = (event, index, value) => {
+    this.setState({
+      sortBy: value
+    })
+  }
+
+  reportFilterMenu (statusFilterValues) {
+    const status = ['Draft', 'Finalized', 'Created By Me']
+    return status.map((name) => (
+      <MenuItem
+        key={name}
+        insetChildren
+        checked={statusFilterValues && statusFilterValues.indexOf(name) > -1}
+        value={name}
+        primaryText={name}
+      />
+    ))
+  }
+
+  reportSortMenu (sortBy) {
+    const labels = [
+      [SORT_BY.DATE_ASCENDING, 'Newest'],
+      [SORT_BY.DATE_DESCENDING, 'Oldest'],
+      [SORT_BY.NAME_ASCENDING, 'Name (A to Z)'],
+      [SORT_BY.NAME_DESCENDING, 'Name (Z to A)']
+    ]
+    return Array.from(labels).map(([sortBy, label], index) => (
```

Comment:
Following the previous change, `Array.from` can now be omitted. It's already an array anyway :smile:

Reply:
@nbiton Thank you for pointing it out! This code is ready for review 😄 😄